app/script/conversation/ConversationRepository.js

```diff
@@ -616,6 +616,13 @@ z.conversation.ConversationRepository = class ConversationRepository {
     this.updateConversations(this.conversations_unarchived());
   }
 
+  updateConversationFromBackend(conversationEntity) {
+    this.conversation_service.get_conversation_by_id(conversationEntity.id).then(conversationData => {
+      this.conversation_mapper.update_properties(conversationEntity, conversationData);
+      this.conversation_mapper.update_self_status(conversationEntity, conversationData);
```

Comment:
I suggest we add the `name` property handling in the `self_update_status` and remove the call the `update_properties`. Those two lines are very confusing I think, it seems each line is doing the same job.

Since this `update_self_status` is ever only called with data from backend, I would also suggest that we rename it `updateFromBackend` (so that it's clear that it makes the translation between properties from the backend and local properties of the entity).
WDYT @bennyn @gregor ?